supplementary/alica_designer_runtime/docker-compose.yml

```diff
@@ -32,7 +32,7 @@ services:
       - OAUTH_APPLICATION_URL=http://localhost:3030
       - NATIVE_MODE=${NATIVE_MODE:-false}
 
-    command: python manage.py runserver 0.0.0.0:8000
+    command: python3 manage.py runserver 0.0.0.0:8000
```

Comment:
This command is run inside docker so maybe shouldn't be changed.  I assume it probably works either way since it probably only has python3 installed but still

Reply:
Sorry, missed this comment, CI was failing that's why I changed it to python3